src/cake.py

```diff
@@ -99,6 +99,12 @@ class Material(object):
     :param lame: tuple with Lame parameter `lambda` and `shear modulus` [Pa]
     :param qk: bulk attenuation Qk
     :param qmu: shear attenuation Qmu
+    :param eta1: transient viscosity (dashpot of the Kelvin-Voigt body;
+                 <= 0 means infinity value) [Pa*s]
+    :param eta2: steady-state viscosity (dashpot of the Maxwell body;
+                 <= 0 means infinity value) [Pa*s]
+    :param alpha: ratio between the effective and the unrelaxed shear modulus
```

Comment:
`alpha` could be confusing because it is often used for P-wave velocity. What about prefixing the new variables with 'v' like visco just that would feel similar to `qk` and `qmu`

Reply:
agreed!